cics-bundle-deploy-reactor-archetype/src/main/resources/archetype-resources/__rootArtifactId__-war/pom.xml

```diff
@@ -12,13 +12,33 @@
   <packaging>war</packaging>
   <name>Example web application</name>
 
-  <dependencies>
+  <dependencyManagement>
+    <dependencies>
       <dependency>
-          <groupId>javax.servlet</groupId>
-          <artifactId>javax.servlet-api</artifactId>
-          <version>3.1.0</version>
-          <scope>provided</scope>
+        <groupId>com.ibm.cics</groupId>
+        <artifactId>com.ibm.cics.ts.bom</artifactId>
+        <!--
+          Change to a version that matches your release of CICS TS.
+          See https://repo1.maven.org/maven2/com/ibm/cics/com.ibm.cics.ts.bom/ for available versions.
+        -->
+        <version>LATEST</version>
```

Comment:
Is this a symbolic that means something, or the user _must_ replace?

Reply:
This is a meta-version that used to be supported in Maven and [is now deprecated](https://cwiki.apache.org/confluence/display/MAVEN/Maven+3.x+Compatibility+Notes#Maven3.xCompatibilityNotes-PluginMetaversionResolution)... but will work. For example, Liberty use `RELEASE` for their archetype. I think I will change this to `RELEASE`, `LATEST` is supposed to be the latest snapshot...